app/yaba.yaml

```diff
@@ -101,6 +106,12 @@ paths:
       operationId: Meta.insert_sites
       tags:
         - sites
+      parameters:
+        - in: query
+          name: status
+          schema:
+            type: boolean
+          description: Status for uploading.
```

Comment:
Same comment here as for above `status` description. The description for `status` can also be updated elsewhere

Reply:
Will make this more clear in the description!